english/story-continues

```diff
@@ -0,0 +1 @@
+The story continues.  You wake up and it was all a dream.  Then, in the corner of your eye you see [a] (http://ancientexplorers.com/wp-content/uploads/sites/6/2015/12/sparkling-blue-universe_131922.jpg)  
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.